test/test-utils.js

```diff
@@ -0,0 +1,90 @@
+/*jshint expr: true*/
+"use strict";
+var chai = require("chai");
+var expect = chai.expect;
+chai.config.includeStack = true;
+var utils = require("../utils");
+
+describe("Utilities testing", function() {
+  it("checks the integrity of the utilities", function() {
```

Comment:
IMO this isn't a useful test. The actual calls matter, while the fact that these functions are declared seems superfluous.